charts/incubator/tauticord/values.yaml

```diff
@@ -0,0 +1,79 @@
+image:
+  repository: tccr.io/truecharts/tauticord
+  pullPolicy: IfNotPresent
+  tag: v3.8.0@sha256:3f834aac482c202963b6882ff78be23d94cc34e29f16a7642010a9db9ee9cdc1
+
+service:
+  main:
+    enabled: false
+    ports:
+      main:
+        enabled: false
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          probes:
+            liveness:
+              enabled: false
+            readiness:
+              enabled: false
+            startup:
+              enabled: false
+          env:
+            TC_DISCORD_BOT_TOKEN: ""
+            TC_DISCORD_SERVER_ID: ""
+            TC_TAUTULLI_KEY: ""
+            TC_TAUTULLI_URL: ""
+            TC_PLEX_PASS: "False"
+            TC_REFRESH_SECONDS: 15
+            TC_TERMINATE_MESSAGE: "Your stream has ended."
+            TC_SERVER_NAME: "Plex"
+            TC_USE_24_HOUR_TIME: "False"
+            # Hide Info
+            TC_HIDE_USERNAMES: "False"
+            TC_HIDE_PLATFORMS: "False"
+            TC_HIDE_PLAYER_NAMES: "False"
+            TC_HIDE_QUALITY: "False"
+            TC_HIDE_BANDWIDTH: "False"
+            TC_HIDE_TRANSCODE: "False"
```

Comment:
Can you try on custom app with lowercase `false` / `true`?
It should work (python should be able to resolve it correctly when reading variables)

Reply:
If it works change all to boolean and merge
if does not work, merge as is

Reply:
Oh ok i was going off that it required string but I'll change it.